metadata.json

```diff
@@ -17,6 +17,10 @@
     {
       "name": "puppetlabs-stdlib",
       "version_requirement": ">= 4.10.0 < 5.0.0"
```

Comment:
Archive requires `puppetlabs-stdlib` with this version string: `>= 4.13.1 < 6.0.0`